packages/vscode-extension/src/handlers.ts

```diff
@@ -884,6 +886,38 @@ async function openMarkdownHandler() {
   }
 }
 
+async function postUpgrade(): Promise<void> {
+  await openUpgradeChangeLogsHandler();
+  await popupAfterUpgrade();
+}
+
+async function popupAfterUpgrade(): Promise<void> {
+  const aadClientSecretFlag = "NeedToSetAADClientSecretEnv";
+  const aadClientSecret = globalStateGet(aadClientSecretFlag, "");
+  if (
+    aadClientSecret !== "" &&
+    workspace.workspaceFolders &&
+    workspace.workspaceFolders.length > 0
+  ) {
+    try {
+      const learnMoreLink = "https://aka.ms/teamsfx-existing-aad-doc";
+      const learnMoreText = "Learn More";
+      const option = { modal: false };
+
+      const outputMsg = `Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system env variable. Read this wiki(${learnMoreLink}) to learn about details.`;
```

Comment:
Please set "AAD_APP_CLIENT_SECRET" = "${aadClientSecret}" in your system environment variable. Read this wiki(${learnMoreLink}) to for more details.